docs/includes/generated_docs/schemas/beta.raw.tpp.md

```diff
@@ -172,14 +172,25 @@ This table contains some historical APCS data.
 <p class="dimension-indicator"><code>many rows per patient</code></p>
```

Comment:
I think it's one row per patient?

Reply:
Oh must be something wrong with my code as I tried to count the number of admissions per person (using `dataset.n_admissions =  isaric_raw.count_for_patient()` and no one had more than one.

Reply:
I think conceptually it's still a many rows per patient table, so i think it's ok that it stays like this. Especially as, currently, there's greater query flexibility for these than for one row per patient tables